src/components/motorcycleForm/form.css

```diff
@@ -0,0 +1,3 @@
+.form {
+  flex: 0 1 30%;
+}
```

Comment:
I would recommend writing most of your CSS styles inside `App.css` rather than creating a file for each component.